packages/frontend/app/services/new-version.js

```diff
@@ -0,0 +1,57 @@
+import Service from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import Ember from 'ember';
```

Comment:
Importing `Ember` is on the way to final deprecation, use `import { isTesting } from '@embroider/macros';` instead to discover this.

Reply:
thanks. replaced. :heavy_check_mark: